profile_library/shelly/Shelly Plus 2PM/model.json

```
@@ -0,0 +1,29 @@
+{
+  "measure_description": "The measured device fw: 1.4.4, Eco Mode enabled, Wifi Client enabled, WiFi AP disabled, BlueTooth disabled. The result represents the average value of measurements collected from the measurement device API for the following scenarios: dumb device/baseline consumption (~4.9W), all relays turned off, one relay turned on",
+  "measure_method": "manual",
+  "measure_device": "Shelly PM Mini gen3",
+  "measure_device_firmware": "20241011-114503/1.4.4-g6d2a586",
+  "measure_settings": {
+    "SAMPLE_COUNT": 200,
+    "SLEEP_TIME": 0.5
+  },
+  "name": "Shelly Plus 2PM",
+  "aliases": [
+    "SNSW-102P16EU"
+  ],
+  "discovery_by": "entity",
+  "device_type": "smart_switch",
+  "only_self_usage": true,
+  "calculation_strategy": "multi_switch",
+  "standby_power": 0.326,
+  "multi_switch_config": {
+    "power": 0.356,
+    "power_off": 0
+  },
```

```suggestion
  "multi_switch_config": {
    "power": 0.356
  },
```

power off can be removed

---

yep. I removed it.